angelsrefining/prototypes/override/refining-override-sorting.lua

```diff
@@ -622,8 +632,28 @@ OV.patch_recipes(
         ["zinc-ore"] = {0, 0, 0, 0, 0.4, 0, 0, 0, 0},
         ["rutile-ore"] = {0, 0, 0, 0, 0.4, 0, 0, 0, 0},
         ["bauxite-ore"] = {0, 0, 0, 0, 0, 0.4, 0, 0, 0},
-        ["manganese-ore"] = {0, 0, 0, 0, 0, ore_exists("platinum-ore") and ore_exists("chrome-ore") and 0.4 or ore_exists("chrome-ore") and 0.2 or 0.4, 0, 0, 0},
-        ["chrome-ore"] = {0, 0, 0, 0, 0, ore_exists("platinum-ore") and 0 or 0.2, ore_exists("platinum-ore") and 0.4 or 0, 0, 0},
+        ["manganese-ore"] = {
+          0,
+          0,
+          0,
+          0,
+          0,
+          ore_exists("platinum-ore") and ore_exists("chrome-ore") and 0.4 or ore_exists("chrome-ore") and 0.2 or 0.4,
+          0,
+          0,
+          0
```

Comment:
to be honest those format changes do not add a lot.

Reply:
I prefer it to be a single line myself as well... it's more readable that way

Reply:
Resolved in https://github.com/Arch666Angel/mods/pull/242/commits/1f4720ccd550e232b805dac945c55d2911314073